### ISSUE TITLE
TVK RedHat Openshift Operator reinstalling web backend every 15 minutes

### ISSUE THREAD

**Issue by jyausm** [triliovault-k8s-issues/triliovault-k8s-issues#30]
## Issue Description
Using RedHat Openshift 4.9
Install TVK Operator from Openshift Operator Hub
After initial successful install, the triliovault web backend pod reinstalls every 15 minutes

### Preconditions

RedHat Openshift 4.9
TVK 2.5.2
Installed into all namespaces
Have tried using multiple projects in RedHat, (new, default, openshift operator)
```

### Steps to reproduce

1. Install TVK RedHat Operator
2. Wait 15 minutes
3. You will see the web-backend in the Operator Details about every 15 minutes
4. View the triliovault web backed pod will show a readiness probe failing
...

```

### Actual and Expected result
We haven't tried any backups or restores since seeing this issue.

**Comment by madhurtrilio** [comment]
Hi @jyausm can you please upgrade to the latest TVK version and check if this is fixed for you.
The latest one available on Openshift is TVK 2.6.7

**Comment by madhurtrilio** [comment]
Closing this as this is resolved
